pinot-core/src/main/java/org/apache/pinot/core/operator/CombineGroupByOperator.java

```diff
@@ -174,8 +173,8 @@ public void runJob() {
           } catch (EarlyTerminationException e) {
             // Early-terminated because query times out or is already satisfied
           } catch (Exception e) {
-            LOGGER.error("Exception processing CombineGroupBy for index {}, operator {}, brokerRequest {}", index,
-                _operators.get(index).getClass().getName(), _brokerRequest, e);
+            LOGGER.error("Caught exception while processing group-by for index: {}, operator: {}, queryContext: {}",
```

Comment:
Perhaps add something to indicate that this is the combine phase? For example, `Caught exception in combine phase of group-by...`?

Reply:
Added